hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/RegionFactoryDefaultSlowTest.java

```diff
@@ -61,4 +65,19 @@ public void testQueryCacheCleanup() {
         assertEquals(numberOfEntities - evictedItemCount, queryRegion.getCache().size());
     }
 
+    @Test
+    public void testUpdate() {
+        insertDummyEntities(1);
+        Session session = sf.openSession();
+        Transaction tx = session.beginTransaction();
+        DummyEntity ent = (DummyEntity) session.get(DummyEntity.class, 0L);
```

Comment:
Or even better.. let the insert method return a list of the id's inserted. This way all this information is directly available instead through some global state.

int[] ids = insertDummyEnties(1)

Or.. simpler:

int id = insertDummyEntity();